catroid/src/org/catrobat/catroid/pocketmusic/note/trackgrid/GridRow.java

```diff
@@ -52,7 +51,7 @@ public int hashCode() {
 		int result = 21;
 		result = 31 * result + noteName.hashCode();
```

Comment:
Thats pretty much how hashCode() should be implemented by the guidelines in the book Effective Java. I like it!